voorhoede/head-start
# feat: remove demo files (hello, geo, social share)

# Changes

- Removes demo API routes and demo SocialShare component.
- Replaces `geo` demo with example in documentation.

# Associated issue

N/A

# How to test

1. Open preview link
2. Verify home and detail page still work without SocialShare component
3. Review new Cloudflare Runtime section in docs

# Checklist

- ~I have added a decision log entry if the change affects the architecture or changes a significant technology~
